src/Model/Generated/Endpoint/DraftPayment.php

```diff
@@ -470,6 +479,28 @@ public function setObject($object)
         $this->object = $object;
     }
 
+    /**
+     * The reference to the object used for split the bill. Can be
+     * RequestInquiry or RequestInquiryBatch
+     *
+     * @return RequestInquiryReference[]
+     */
+    public function getRequestReferenceSplitTheBill()
```

Comment:
No return type generated?

Reply:
You mean type hint ? 

No, once we update the min required php version. As these fields can be null.

Reply:
No, return type.. Doc does not specify this function can return `null`. So which one is wrong?

Reply:
@OGKevin

Reply:
Yea I meant return type hint :P. Well the thing is, the generator can not know if these response field will always be present and therefore add `|null` as the definitions do no define nullable. If not we would need to add `|null` to all response fields 🤔

Reply:
Hmm I see, let's leave it like this then. Sounds like an unnecessary refactor.